models/public/resnet-18-pytorch/model.yml

```
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Resnet 18 is image classification model pretrained on ImageNet dataset. This
+  is PyTorch implementation based on architecture described in paper "Deep Residual
+  Learning for Image Recognition" <https://arxiv.org/pdf/1512.03385> in TorchVision
+  package (see here <https://github.com/pytorch/vision>).
+
+  The model input is a blob that consists of a single image of "1x3x224x224" in
+  RGB order.
+
+  The model output is typical object classifier for the 1000 different classifications
+  matching with those in the ImageNet database.
+task_type: classification
+files:
+  - name: resnet18-5c106cde.pth
+    sha256: 5c106cde386e87d4033832f2996f5493238eda96ccf559d1d62760c4de0613f8
+    size: 46827520
+    source: https://download.pytorch.org/models/resnet18-5c106cde.pth
+framework: pytorch
+conversion_to_onnx_args:
+  - --model-name=resnet-18
```

according to torchvision code https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py#L6
```suggestion
  - --model-name=resnet18
```